keyboards/kbdfans/kbd67/mkiirgb/keymaps/mikefightsbears/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 MechMerlin
```

Comment:
This file can be deleted, as it's not doing anything.

Reply:
removed